flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/dql/SqlRichDescribeModel.java

```diff
@@ -41,7 +41,7 @@ public class SqlRichDescribeModel extends SqlCall {
     public static final SqlSpecialOperator OPERATOR =
             new SqlSpecialOperator("DESCRIBE MODEL", SqlKind.OTHER);
     protected final SqlIdentifier modelNameIdentifier;
-    private boolean isExtended;
+    private final boolean isExtended;
```

Comment:
Is this change needed?

Reply:
not strictly needed, but the IDE preferred it this way, so i wanted to leave the codebase cleaner than I found it